src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -493,3 +494,38 @@ Managing an email queue is an easy task if you follow this procedure:
 In case, you need to clear an email queue because there are only spam or trash emails in the email queue you have this procedure to follow:
 
 - All mails from the given mail queue will be deleted with [Clearing a mail queue](manage-webadmin.html#Clearing_a_mail_queue).
+
+## Deleted Messages Vault
+
+Deleted Messages Vault is an interesting feature that will help James users have a chance to:
```

Comment:
other features are not interesting? :trollface:

Reply:
I'm not saying that :trollface: